src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.props

```diff
@@ -74,6 +74,23 @@ Copyright (c) .NET Foundation. All rights reserved.
     <None Remove="**\*.razor" />
   </ItemGroup>
 
+  <!-- Properties related to handling static web assets in the package  -->
+  <PropertyGroup>
+    
+    <!-- Used to indicate that a $(PackageId).props importing Microsoft.AspNetCore.StaticWebAssets.props file should be
+         automatically generated and included in the package under build\$(PackageId).props -->
+    <StaticWebAssetsEnableProjectBuildPropsFileGeneration>true</StaticWebAssetsEnableProjectBuildPropsFileGeneration>
+
+    <!-- Used to indicate that a $(PackageId).props importing build\$(PackageId).props file should be
+         automatically generated and included in the package under buildMultiTargeting\$(PackageId).props -->
+    <StaticWebAssetsEnableProjectBuildMultiTargetingPropsFileGeneration>true</StaticWebAssetsEnableProjectBuildMultiTargetingPropsFileGeneration>
+
+    <!-- Used to indicate that a $(PackageId).props importing buildMultiTargeting\$(PackageId).props file should be
+         automatically generated and included in the package under buildTransitive\$(PackageId).props -->
+    <StaticWebAssetsEnableProjectBuildTransitivePropsFileGeneration>true</StaticWebAssetsEnableProjectBuildTransitivePropsFileGeneration>
+  
+  </PropertyGroup>
```

Comment:
Need to honor these properties yet.

There will be a separate PR to add more configuration options once this one and the publish one goes in. We likely want to allow users do the following:
* Turn off all static web assets feature as a whole
* Disable the generation of pack target files (and auto packing) as a whole (and for individual target files)
* Include static web assets as regular content in addition to as static web assets.